src/estree.ts

```diff
@@ -169,7 +169,10 @@ export type Expression =
   | UnaryExpression
   | UpdateExpression
   | YieldExpression;
-export type ForInitialiser = Expression | VariableDeclaration;
+export type ForInitializer = Expression | VariableDeclaration;
+// TODO: Misspelled, remove this in v7
+// spellchecker: disable-next-line
+export { ForInitializer as ForInitialiser };
```

Comment:
Shuld we consider removing this as breaking change?

Reply:
"Initializer" is US spelling, which is correct for Kenny.

Reply:
cspell should have config for US and British.

Reply:
This PR is changing "Initialiser" to "Initializer", I believe both US and British use "Initializer", I'm adding this alias, since it was exported.

Reply:
Oh I missed that.